demos/EventBasedRisk/job.ini

```diff
@@ -1,65 +1,54 @@
 [general]
-
-description = Probabilistic Event-Based Hazard Demo
+description = Event-Based Risk Demo (Nepal)
 calculation_mode = event_based_risk
 concurrent_tasks = 16
-random_seed = 1024
-
-[risk]
-master_seed = 42
-asset_hazard_distance = 20
-loss_curve_resolution = 20
-
-conditional_loss_poes = 0.01
+random_seed = 42
 
-insured_losses = True
-
-specific_assets = a1 a2 a3 a4 a5 a6
-
-structural_vulnerability_file = structural_vulnerability_model.xml
-nonstructural_vulnerability_file = nonstructural_vulnerability_model.xml
-loss_ratios = {'structural': [0.1, 0.2, 0.3, 0.5, 0.8, 1.0], 'nonstructural': [0.01, 0.02, 0.1, 0.2, 0.5]}
-
-[geometry]
-region_constraint = 78.0 31.5,89.5 31.5,89.5 25.5,78 25.5
+[exposure]
 exposure_file = exposure_model.xml
 
-[logic_tree]
-
-number_of_logic_tree_samples = 0
+[boundaries]
+region_constraint = 78.0 31.5, 89.5 31.5, 89.5 25.5, 78.0 25.5
 
 [erf]
-
-# km
 rupture_mesh_spacing = 10
 width_of_mfd_bin = 0.5
-# km
 area_source_discretization = 10
 
 [site_params]
-
 reference_vs30_type = measured
 reference_vs30_value = 760.0
 reference_depth_to_2pt5km_per_sec = 5.0
 reference_depth_to_1pt0km_per_sec = 100.0
 
-[calculation]
+[vulnerability]
+structural_vulnerability_file = structural_vulnerability_model.xml
+nonstructural_vulnerability_file = nonstructural_vulnerability_model.xml
 
+[logic_trees]
 source_model_logic_tree_file = source_model_logic_tree.xml
 gsim_logic_tree_file = gmpe_logic_tree.xml
-# years
-investigation_time = 50.0
+number_of_logic_tree_samples = 0
 
+[hazard_calculation]
+investigation_time = 1.0
 truncation_level = 3
-# km
 maximum_distance = 200.0
+ses_per_logic_tree_path = 1000
 
-ses_per_logic_tree_path = 200
-
-[output]
+[risk_calculation]
+master_seed = 42
+asset_hazard_distance = 20
+loss_curve_resolution = 20
+conditional_loss_poes = 0.01
+insured_losses = True
+specific_assets = 
 
-export_dir = /tmp
+[outputs]
 ground_motion_fields = false
-hazard_curves_from_gmfs = true
-mean_hazard_curves = true
+hazard_curves_from_gmfs = false
+mean_hazard_curves = false
 quantile_hazard_curves =
```

Comment:
Michele, better if we change it to true. The idea is to generate at least one map.